### PR TITLE
Update first_contribution.rst

### PR DESCRIPTION
**Pull request by janneke-qgis**
clarified that the "fix me" button is in the footer of the page - I never thought to look in the footer for it.

### CODE REVIEW

File: source/docs/documentation_guidelines/first_contribution.rst [hunk 1]
```diff
@@ -92,7 +92,7 @@ Alternative 2: Using the ``Fix Me`` shortcut
 The QGIS project provides an easy way to reach source file from online documentation.
 Indeed, instead of browsing the source files in GitHub to find the one that suits
 the issue, or if you find an issue while reading the manuals, you simply have to
-click the "Fix Me" link at the bottom of the page to open its source file in Edit mode.
+click the "Fix Me" link at the bottom of the footer of the page to open its source file in Edit mode.
```
Comment (DelazJ) [inline source/docs/documentation_guidelines/first_contribution.rst]: `at the bottom of the footer of the page` is that not too... deep?

Comment (janneke-qgis) [reply source/docs/documentation_guidelines/first_contribution.rst]: Not sure how else to write it - but as long as it is clear that there is useful information in the footer it's fine. Maybe just replace "at the bottom" with "in the footer"?

Comment (DelazJ) [reply source/docs/documentation_guidelines/first_contribution.rst]: yes

**Comment by DelazJ** [comment]
Thanks @janneke-qgis

**Pull request merged**